content/rs/administering/creating-databases/_index.md

```diff
@@ -141,24 +141,22 @@ after the database is created.
             can use [Multi-key commands]({{< relref "/rs/concepts/high-availability/clustering.md" >}})
             without the limitations.
 
-- [**Data eviction policy**]({{< relref "/rs/administering/database-operations/eviction-policy.md" >}}) -
-    By default, when the total size of the database reaches its memory limit the database evicts keys
-    according to the least recently used keys out of all keys with an "expire" field set
-    in order to make room for new keys. You can select a different data eviction policy.
-
-- [**Replica Of**]({{< relref "/rs/administering/creating-databases/create-active-passive.md" >}}) -
-    You can make this database a repository for keys from other databases.
-
-- [**TLS**]({{< relref "/rs/administering/designing-production/security/tls-configuration.md" >}}) -
-    You can require TLS encryption and authentication for all communications,
-    TLS encryption and authentication for Replica Of communication only, and TLS authentication for clients.
-
-- [**Periodic backup**]({{< relref "/rs/administering/import-export/database-backup.md" >}}) -
-    You can configure periodic backups of the database, including the interval and backup location parameters.
-
-- [**Alerts**]({{< relref "/rs/administering/monitoring-metrics/_index.md#database-alerts" >}}) -
-    You can select alerts to show in the database status and configure their thresholds.
-    You can also select to send the alerts by email to [relevant users]({{< relref "/rs/administering/designing-production/access-control/_index.md" >}}).
+    - [**OSS Cluster API**]({{< relref "/rs/administering/designing-production/networking/using-oss-cluster-api.md" >}}) - {{< embed-md "oss-cluster-api-intro.md"  >}}
+    - [**Data eviction policy**]({{< relref "/rs/administering/database-operations/eviction-policy.md" >}}) -
+        By default, when the total size of the database reaches its memory limit the database evicts keys
+        according to the least recently used keys out of all keys with an "expire" field set
+        in order to make room for new keys. You can select a different data eviction policy.
+    - [**Replica Of**]({{< relref "/rs/administering/creating-databases/create-active-passive.md" >}}) -
+        You can make this database a repository for keys from other databases.
```

Comment:
@bmansheim 
Replica Of description - 
In order to create an Active-Passive replicated database (also known as Replica Of), which gives applications read-only access to replicas of the data that are hosted in different geographical locations or clusters, you can activate the Replica Of option and configure the wanted databases to replicate.

Or something similar.

Reply:
How's this?
You can give applications read-only access to replicas of the data that are hosted in different geographical locations or clusters. This is called Active-Passive or Replica Of. To configure the replication, enable Replics Of and specify the databases to replicate to this database.

Reply:
@orshwartzredislabs Can you comment on my suggestion?

Reply:
@bmansheim Looks good!

Reply:
@bmansheim did this description go anywhere? I want to emphasize and make sure we don't mislead users - Active-passive (replica of) does not limit the access to it to read only. It's up to the application to make sure it does only reads. A user can leverage ACL to assist him. But out-of-the-box replicaOf does not limit access.
_"You can give applications read-only access to replicas of the data that are hosted in different geographical locations or clusters. This is called Active-Passive or Replica Of. To configure the replication, enable Replics Of and specify the databases to replicate to this database."_